app.json

```diff
@@ -41,6 +41,6 @@
     }
   ],
   "scripts": {
-    "postdeploy": "python manage.py review_app_admin"
+    "postdeploy": "python manage.py load_fake_data && python manage.py review_app_admin"
```

Comment:
Yes, because post-deploy scripts are running [after the release phase](https://devcenter.heroku.com/articles/release-phase#review-apps-and-postdeploy-script).

If we move the `load_fake_data` as a release step, it will delete and regenerate new fake stuff every time a push is made to a PR. It's probably not what we want (and it will delete the admin account).